### DISCUSSION TITLE
spelling with Abdullah

### DISCUSSION THREAD

**Discussion by digger-yu** [streetsidesoftware/cspell-dicts#2136] [Q&A]
![image](https://user-images.githubusercontent.com/55081697/234450044-15d51f1b-44ca-4c6a-beb1-0c5e1ec53315.png)

King Abdullah University of Science and Technology

**Comment by Jason3S** [comment]
@digger-yu,

There are many universities and companies that haven't been added to a dictionary yet. Please add it to [dictionaries/src/companies.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/companies/src/companies.txt). Just `edit`  the file and make a PR.
<img width="211" alt="image" src="https://user-images.githubusercontent.com/3740137/234490345-f0225809-d99d-4a30-a44d-8804b55c43c8.png">

**Reply by digger-yu** [reply]
Submitted a PR done
